### ISSUE TITLE
Export Movie Report

### ISSUE THREAD

**Issue by psouza4** [psouza4/mediacentermaster#1383]
Originally reported on Google Code with ID 1386

```
I originally sent this in the forums, but it looks to be more of a feature request.
 When exporting the media library, the report has no way to add wanted fields. Currently
the video and audio fields are not included.

Request to make the report include them or maybe if the report can be tailored made
by selecting wanted fields and then exporting it.

Thanks
```

Reported by `darooster10` on 2013-06-07 22:35:28

**Comment by pliesj** [comment]
I totally agree that a customized HTML export file would be very useful.  I have over 9,000 movies in my collection, and I desire more information to be presented for each film, especially since I have many instances of different films having the same title that were released in the same year or within a year or two of one another.

Specifically, I would like the export file to be able to include at least the COUNTRY and the DIRECTOR of the film.  Since many film buffs think of films in terms of who directed the picture, this information would be extremely useful.
